build/myget-release.yml

```diff
@@ -11,6 +11,8 @@ variables:
     value: '2.2.105'
   - name: 'Project'
     value: 'Arcus.EventGrid'
+  - name: 'Package.Version.ManualTrigger'
```

Comment:
Can we still change it when we queue it or not?

Reply:
I can't find anything in the docs that says we can't; so I guess it's the same as setting a build variable. How could we verify this?

Reply:
1. Go to Azure DevOps
2. Go to pipelines
3. Go to your build
4. Click "Run Pipeline"
5. Can you specify a version there, similar to our old builds?

Reply:
I can specify one with the same name

Reply:
Do you have to provide the variable name yourself or can you just change the default value?

Reply:
Provide it yourself...

Reply:
Ah, found it; I have to manually set the vairable in the editor since variables at queue time cannot be specified in the variables block in the YAML file.

Reply:
So I removed the variable in the script, but added it to the variables in the editor (which now shows when you run the build).

Reply:
Perfect, thank you!